collect-structure.rst

```diff
@@ -0,0 +1,98 @@
+***********************************************************
+Structure of ODK Collect directory in Android file system
+***********************************************************
+
+After installation of ODK collect, you can try out the app by :ref:`downloading blank example forms <in-app-get-blank-forms>`, as soon as you download a blank form you would be able see odk directory in the internal storage or external storage depending on the settings of your android device. The structure of the directory is explained below.
+
+.. _directory-structure:
+
+Directory Structure
+----------------------
+
+::
+
+ odk
+ ├── forms
+ │   ├── Sample-form-media
+ │   └── Sample form.xml
+ │  
+ ├── instances
+ │   └── formID_yyyy-mm-dd_hh-MM-ss
+ │       └── formID_yyyy-mm-dd_hh-MM-ss.xml
+ │  
+ ├── layers
+ │   └── Sample layer
+ │        └── file.mbtiles
+ │
+ └── metadata  
+	├── forms.db
+	├── forms.db-journal
+	├── instances.db
+	└── instances.db-journal
+	
+Inside :file:`ODK` directory, there are four subdirectories which are as follows:
+
+.. image:: /img/collect-structure/directory.*
+  :alt: Image showing folders inside odk directory
+  :class: device-screen-vertical
+
+.. _forms-directory:
+
+:guilabel:`forms` Directory
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: /img/collect-structure/sample-form.*
+  :alt: Image showing forms and Sample form-media directory
+  :class: device-screen-vertical
+
+:file:`forms` folder contains all the forms, fetched from aggregate server, also forms designed by users do not have to be uploaded to Aggregate for them to be used. They can be manually added to the forms directory. The :file:`Sample form-media` directory holds :file:`csv` files.
```

Comment:
- Contains blank forms, whether downloaded from an Aggregate server, google drive, or whatever. 
 - If are uploading forms to the device manually, this is where they go.


> The :file:`Sample form-media` directory holds :file:`csv` files.

This statement is confusing if you don't already know what it means. The fact that this also contains folders for form media needs to be made explicit here, not just mentioned in reference to an example.